src/Applications/LBAF.py

```diff
@@ -132,6 +132,9 @@ def __init__(self):
         # Output directory
         self.output_dir = None
 
+        # Output file steam
+        self.output_file_stem = None
```

Comment:
I am assuming this will result in a runtime error if the value is not provided, right?
Maybe we should have a default value for when the configuration file does not specify the output file stem.

Reply:
If the `output_file_stem` in conf file is not provided, then it will use the old behaviour/naming convention